### ISSUE TITLE
Does Nanomsg support 2.6 kernel? Does nng support it?

### ISSUE THREAD

**Issue by lwsli** [nanomsg/nanomsg#1113]
Our application may be running on the 2.6 kernel and we are considering using nanomsg.
Communication methods such as ipc and inproc may be used.

**Comment by gdamore** [comment]
Yes.  It should be fine.  Although that kernel is by now ancient.
